.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discussion
+    url: https://github.com/nats-io/nats-server/discussions
```

Comment:
Not `nats-server`, or is the intent that we all use one discussion?

Reply:
That was a copy paste error, fixing.